packages/shorebird_cli/lib/src/commands/patch_command.dart

```diff
@@ -398,6 +398,7 @@ If you believe you're seeing this in error, please reach out to us for support a
         archMetadata.path,
         'libapp.so',
       );
+      logger.detail('Creating artifact for $patchArtifactPath');
```

Comment:
@felangel we've been using `[DOMAIN]` prefixes elsewhere, but I'm not sure what a sensible one would be for the top-level command. Maybe we don't need one? Thoughts?

Reply:
Yeah I guess we don't need one here